onchain/rollups/test/foundry/dapp/Application.t.sol

```diff
@@ -441,79 +442,6 @@ contract ApplicationTest is TestBase {
         _executeVoucher(voucher, proof);
     }
 
-    function testWithdrawEtherContract(
-        uint256 value,
-        address notApplication
-    ) public {
-        vm.assume(value <= address(this).balance);
-        vm.assume(notApplication != address(_app));
-        address receiver = address(new EtherReceiver());
```

Comment:
Can remove the declaration of `EtherReceiver` contract in this test contract, since it's not used anywhere else